tests/python/unittest/test_random.py

```diff
@@ -844,6 +844,50 @@ def testLarge(data, repeat):
     testLarge(mx.nd.arange(0, 100000).reshape((10, 10000)), 10)
     testLarge(mx.nd.arange(0, 100000).reshape((10000, 10)), 10)
 
+@with_seed()
+def test_randint():
+    dtypes = ['int32', 'int64']
```

Comment:
can we also include 'uint8'','int8' in the test?

Reply:
std:uniform_int_distribution doesn't support uint8 and int8. Hence I didn't include that

Reply:
updated the front-end api doc with the same